samples/cellular/sms/src/main.c

```diff
@@ -78,12 +80,29 @@ int main(void)
 		return 0;
 	}
 
+	/* Setting SMS service center number if set into the configuration. */
+	if (sizeof(CONFIG_SMS_SERVICE_CENTER_NUMBER) > 1) {
+		int err = nrf_modem_at_cmd(
+				response, sizeof(response),
+				"AT+CSCA=\"%s\",91",
+				CONFIG_SMS_SERVICE_CENTER_NUMBER);
+		if (err) {
+			printk("Setting service center number %s failed, err=%d\n",
+				CONFIG_SMS_SERVICE_CENTER_NUMBER, err);
+		} else {
+			printk(
+				"Setting service center number %s succeeded\n",
+				CONFIG_SMS_SERVICE_CENTER_NUMBER);
```

Comment:
```suggestion
			printk("Setting service center number %s succeeded\n",
			       CONFIG_SMS_SERVICE_CENTER_NUMBER);
```

Reply:
The old way would be the accurate if parameters are on separate lines when they don't fit into the same line as function name but I know this will make people feel uncomfortable 😄